src/host/usbh.c

```diff
@@ -509,7 +509,7 @@ void tuh_task_ext(uint32_t timeout_ms, bool in_isr)
       default: break;
     }
 
-#if CFG_TUSB_OS != OPT_OS_NONE && CFG_TUSB_OS != OPT_OS_PICO
+#if CFG_TUSB_OS != OPT_OS_NONE && CFG_TUSB_OS != OPT_OS_PICO && CFG_TUSB_OS != OPT_OS_FREERTOS
```

Comment:
please revert this change, since we want application to be able to share usb with other background if needed

Reply:
I have also reverted this commit but I would like to discuss this further with you, I will create a discussion topic for it if you don't mind?

Cheers,

Pete